Framework/Geometry/src/Math/Quadrilateral.cpp

```diff
@@ -156,5 +156,16 @@ ConvexPolygon Quadrilateral::toPoly() const {
   return ConvexPolygon(points);
 }
 
+/// Shifts the vertexes in a clockwise manner
+void Quadrilateral::shiftVertexesClockwise() {
+
```

Comment:
As a clean-up task. A better function would be one that simply rearranged the vertexes to be in the right order for the intersection algorithms. Something like `orderClockwise()` and `isOrderedClockwise()` may be good names. I would also suggest adding a static public constructional method, called `makeClockwise()`, which called the existing constructor and returned the created object, but did everything necessary to put the vertexes in the right place.